src/main/java/com/be3c/sysmetic/domain/strategy/controller/ExcelController.java

```diff
@@ -0,0 +1,83 @@
+package com.be3c.sysmetic.domain.strategy.controller;
+
+import com.be3c.sysmetic.domain.strategy.service.ExcelService;
+import com.be3c.sysmetic.global.common.response.APIResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.InputStream;
+
+@RestController
+@RequestMapping("/api/excel")
+@RequiredArgsConstructor
+public class ExcelController {
+
+    final ExcelService excelService;
+
+
+    @PostMapping("/upload")
+    public ResponseEntity<APIResponse<String>> uploadExcel(
+            @RequestParam("file") MultipartFile file,
+            @RequestParam("strategyId") Long strategyId) {
+
+        excelService.uploadExcel(file, strategyId);
+
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(APIResponse.create("엑셀 파일 입력 완료"));
+    }
+
+
+    @GetMapping("/download/daily")
+    public ResponseEntity<APIResponse<InputStreamResource>> downloadDailyExcel(
+            @RequestParam("strategyId") Long strategyId) {
```

Comment:
```suggestion
    @GetMapping("/download/daily/{strategyId}")
    public ResponseEntity<APIResponse<InputStreamResource>> downloadDailyExcel(
            @PathValiable Long strategyId) {
```
이렇게 바꾸는 것도 좋아 보입니다
[REST API URI Naming Conventions and Best Practices](https://restfulapi.net/resource-naming/)